src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -366,7 +366,7 @@ private InitValueView getInitValue(MethodTransformerContext context, InitCodeNod
         default:
           throw new UnsupportedOperationException("unexpected entity name type");
       }
-    } else if (initValueConfig.hasFormattingConfig()) {
+    } else if (initValueConfig.hasFormattingConfig() && !item.getType().isRepeated()) {
```

Comment:
It's related to #1085. It doesn't fix that issue, but it leads to still incorrect, but a bit more reasonable, generated code.
The problem at now is that the "zero-value" for a repeated resource-name is a list of string, rather than a list of the strongly-typed resource name.